ports/atmel-samd/boards/cp32-m4/mpconfigboard.h

```diff
@@ -1,5 +1,5 @@
 #define MICROPY_HW_BOARD_NAME "CP32-M4"
-#define MICROPY_HW_MCU_NAME "samd51j20"
+#define MICROPY_HW_MCU_NAME "samd51j20A"
```

Comment:
Don't know why the 'A' was added , in uppercase? (I was off the livestream after a while.)

Reply:
It is from: https://github.com/adafruit/circuitpython/pull/8414/files#diff-a7668631002dd4a35f2f0e0802af304e1dda1754d0fd114d4f1996ddabe15150L2

I'll remove it.